app/experimenter/experiments/tests/api/v3/test_serializers.py

```diff
@@ -156,3 +156,21 @@ def test_serializer_creates_changelog_for_updates(self):
                 changed_values=changed_values,
             ).exists()
         )
+
+    def test_serializer_returns_errors_for_non_alpha_numeric_name(self):
+
+        data = {
+            "name": "!!!!!!!!!!!!!!!",
```

Comment:
I should probably just verify on Experimenter Stage using un-rapid experiments, but... Do we have somewhere that verifies that a slug isn't already used? Or do name collisions on slugs not matter?

Reply:
oooh you're right... should check for slug uniqueness as well

Reply:
Good catch @pdehaan let's do that as a followup: https://github.com/mozilla/experimenter/issues/2959